RELEASING.md

```diff
@@ -47,6 +47,7 @@ For Squares, membership is managed through the `Workflow Swift Owners` registry
     bundle exec pod trunk push WorkflowSwiftUI.podspec --synchronous
     bundle exec pod trunk push WorkflowCombine.podspec --synchronous
     bundle exec pod trunk push WorkflowCombineTesting.podspec --synchronous
+    bundle exec pod trunk push ViewEnvironment.podspec --synchronous
```

Comment:
These are ordered by dependency — you'll have to push this one first (or at least before `WorkflowUI`).

Reply:
Good catch, [done](https://github.com/square/workflow-swift/pull/185/commits/f69e7a790ef09813a77d4fbbc5dc716f8d8ab885)!